mm2src/mm2_main/src/docker_tests.rs

```diff
@@ -1134,6 +1121,134 @@ mod docker_tests {
         block_on(mm_alice.stop()).unwrap();
     }
 
+    #[test]
+    fn test_watcher_node() {
+        let (_ctx, _, bob_priv_key) = generate_utxo_coin_with_random_privkey("MYCOIN", 100.into());
+        generate_utxo_coin_with_privkey("MYCOIN1", 100.into(), &bob_priv_key);
+        let (_ctx, _, alice_priv_key) = generate_utxo_coin_with_random_privkey("MYCOIN1", 100.into());
+        generate_utxo_coin_with_privkey("MYCOIN", 100.into(), &alice_priv_key);
+        let watcher_priv_key = *SecretKey::new(&mut rand6::thread_rng()).as_ref();
+
+        let coins = json! ([
+            {"coin":"MYCOIN","asset":"MYCOIN","txversion":4,"overwintered":1,"txfee":1000,"protocol":{"type":"UTXO"}},
+            {"coin":"MYCOIN1","asset":"MYCOIN1","txversion":4,"overwintered":1,"txfee":1000,"protocol":{"type":"UTXO"}},
+        ]);
+
+        let alice_conf = Mm2TestConf::seednode(&format!("0x{}", hex::encode(alice_priv_key)), &coins, true, false).conf;
```

Comment:
This `true, false` arguments sequence is already confusing a bit, and can become even more confusing if we add more arguments. Please add `seednode_using_watchers` and use it instead.

Reply:
Yes I had to make a decision between too many arguments and too many helper functions, and I went with too many arguments. I'll change it.